demisto_sdk/commands/run_cmd/runner.py

```diff
@@ -106,12 +106,16 @@ def run(self):
 
     def _get_playground_id(self):
         """Retrieves Playground ID from the remote Demisto instance."""
-        playground_filter = {"filter": {"type": [9]}}
-        answer = self.client.search_investigations(filter=playground_filter)
+
+        def playground_filter(page: int = 0):
+            return {"filter": {"type": [9], "page": page}}
+
+        answer = self.client.search_investigations(filter=playground_filter())
         if answer.total == 0:
             raise RuntimeError("No playgrounds were detected in the environment.")
-        playgrounds = answer.data
-        if len(playgrounds) > 1:
+        elif answer.total == 1:
+            result = answer.data[0].id
```

Comment:
Did you check this fix?

Reply:
I couldn't check the exact case but did the best I could.